src/Sulu/Component/Rest/ListBuilder/Metadata/Resources/schema/list-2.0.xsd

```diff
@@ -135,6 +135,11 @@
         <xs:attribute name="distinct" type="xs:boolean"/>
     </xs:attributeGroup>
 
+    <xs:attributeGroup name="countPropertyAttributeGroup">
```

Comment:
/cc @sulu/core-team think we need to create a new .xsd when else it would autocomplete distinct in old versions where distinct does not exist. Or what do you think @sulu/core-team u?

Reply:
It is just a minor DX issue and doesn't blow up in any way, right? If we create a new XSD for every single attribute we add, it's going to be very hard to maintain :confused: Therefore I would not do it.